application/application.go

```diff
@@ -0,0 +1,101 @@
+package application
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/josephroberts/edge-node-manager/config"
+	"github.com/josephroberts/edge-node-manager/database"
+	"github.com/josephroberts/edge-node-manager/device"
+	"github.com/josephroberts/edge-node-manager/helper"
+	"github.com/josephroberts/edge-node-manager/proxyvisor"
+)
+
+type Application struct {
+	UUID       string
+	DeviceType *device.Type
+}
+
+func (a *Application) Process() error {
+	log.Printf("-----------------------------------------------------------------\r\n")
+	log.Printf("Application UUID: %s\r\n", a.UUID)
+	log.Printf("Application device type: %s\r\n", a.DeviceType.Device)
+	log.Printf("Application radio type: %s\r\n", a.DeviceType.Radio)
+
+	// Get latest firmware .zip and commit
+	application, commit, err := helper.GetApplication(config.GetPersistantDirectory(), a.UUID)
+	if err != nil {
+		log.Printf("No application location or commit found\r\n")
```

Comment:
You're swallowing the error here and not outputting it anywhere or returning it despite the function signature having an `error` return.

If it's meant to be possible for these to not be retrieve,d then this shouldn't be an error, rather return nil or maybe make the `err` field be a 'found' binary or something.

If you really do want to swallow the error it's better to make a comment explicitly saying that because otherwise it looks like you have an error condition you've accidentally missed here (as a maintainer), this kind of thing is where a comment can be super-useful.